src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -216,6 +235,24 @@ public void start(Map<String, String> properties) {
     running.set(true);
   }
 
+  private List<Map<String, String>> possibleTablePartitions(String table) {
+    TableId tableId = dialect.parseTableIdentifier(table);
+
+    String fqn = ExpressionBuilder.create().append(tableId, false).toString();
+    Map<String, String> partitionWithFqn = new HashMap<>();
+    partitionWithFqn.put(JdbcSourceConnectorConstants.TABLE_NAME_KEY, fqn);
+    partitionWithFqn.put(
+        JdbcSourceConnectorConstants.OFFSET_PROTOCOL_VERSION_KEY,
+        JdbcSourceConnectorConstants.PROTOCOL_VERSION_ONE
+    );
+
```

Comment:
How about adding a comment:

    // Define source partition map for original offset protocol (V0) that uses the table name